### PR TITLE
Feat/improved questionnaire

### PR DESCRIPTION
**Pull request by karstenlenz**
Please review my code.

Story: https://github.com/karstenlenz/capstone-project/issues/26

### CODE REVIEW

File: src/Questionnaire/Questionnaire.js [hunk 1]
```diff
@@ -15,44 +19,67 @@ export default function Questionnaire({ testIds }) {
...
+
   useEffect(() => {
-    questionRound === 2 && history.push(resultUrl)
-  }, [questionRound, resultUrl, history])
+    resultUrl.length === 12 && history.push(resultUrl)
+  }, [resultUrl, history])
 
   return (
     <>
       <img alt="" src="/img/questionnaire-intro.svg" />
```
Comment (mpagels) [inline src/Questionnaire/Questionnaire.js:31]: could use as ReactComponant, too

File: src/Questionnaire/Questionnaire.js [hunk 2]
```diff
@@ -70,3 +97,15 @@ const ButtonRow = styled.section`
   display: flex;
   justify-content: space-between;
 `
+
+const H2Styled = styled.h2`
+  margin-top: -10px;
+`
+const QuestionnaireButton = styled(Button)`
+  ${(props) =>
+    props.borderColor ? 'border: 2px solid ' + props.borderColor + ';' : ''}
```
Comment (mpagels) [inline src/Questionnaire/Questionnaire.js:106]: Could this work?
```suggestion
  ${(props) =>
    props.borderColor && 'border: 2px solid ' + props.borderColor + ';'}
```
do you need the " '' "

**Review by mpagels — COMMENTED** [review]
Incredible job. Found some tiny things.
I noticed that you didn't used PropTypes. Consider it to add this to your project ;-)

Again, very good job.

File: src/Questionnaire/Questionnaire.js (hunk 2 above)
Comment (karstenlenz) [reply src/Questionnaire/Questionnaire.js:111]: No, you are right, I use too many ternaries :). I changed it to you suggestion!

**Pull request merged**